<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StaffPanel</class>
 <widget class="QMainWindow" name="StaffPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1250</width>
    <height>940</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1024</width>
    <height>768</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Welcome to Staff Panel</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/new/images/icons/logoicon.ico</normaloff>:/new/images/icons/logoicon.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">#centralwidget{
background-image: url(:/new/images/Images/a72082c00f056ac229e10ffcbdac9059.jpg);}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QGridLayout" name="gridLayout_6">
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="0" column="0">
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="inst_name">
            <property name="styleSheet">
             <string notr="true">font: 75 24pt &quot;MS Shell Dlg 2&quot;;
color: rgb(0, 85, 255);
qproperty-alignment: AlignCenter;</string>
            </property>
            <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;&lt;span style=&quot; font-weight:600; color:#0055ff;&quot;&gt;Institution Name&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="1" column="0">
         <layout class="QGridLayout" name="gridLayout_5">
          <item row="0" column="0">
           <widget class="QLabel" name="label_10">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="inst_address">
            <property name="styleSheet">
             <string notr="true">font: 75 24pt &quot;MS Shell Dlg 2&quot;;
color: rgb(0, 85, 255);
qproperty-alignment: AlignCenter;</string>
            </property>
            <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;&lt;span style=&quot; font-weight:600; color:#0055ff;&quot;&gt;Address&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QLabel" name="label_11">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="welcomeStaff">
        <property name="styleSheet">
         <string notr="true">color: white; 
font: 22pt &quot;Imprint MT Shadow&quot;;

</string>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:26pt; font-weight:600; color:#0055ff;&quot;&gt;Welcome to Staff Panel&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QGroupBox" name="staffTask">
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="title">
       <string>What do you want to do?</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="QSplitter" name="splitter">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <widget class="QPushButton" name="addStudent">
            <property name="font">
             <font>
              <family>MS Shell Dlg 2</family>
              <pointsize>10</pointsize>
              <weight>50</weight>
              <italic>false</italic>
              <bold>false</bold>
             </font>
            </property>
            <property name="mouseTracking">
             <bool>true</bool>
            </property>
            <property name="toolTip">
             <string>Add Student Records</string>
            </property>
            <property name="styleSheet">
             <string notr="true">
border-image: url(:/new/images/icons/add.png);
</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
           <widget class="QLabel" name="label">
            <property name="styleSheet">
             <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;
color:red;</string>
            </property>
            <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:14pt; font-weight:600;&quot;&gt;Add Student Records&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
           </widget>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QSplitter" name="splitter_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <widget class="QPushButton" name="viewStudent">
            <property name="font">
             <font>
              <family>MS Shell Dlg 2</family>
              <pointsize>10</pointsize>
              <weight>50</weight>
              <italic>false</italic>
              <bold>false</bold>
             </font>
            </property>
            <property name="mouseTracking">
             <bool>true</bool>
            </property>
            <property name="toolTip">
             <string>View Student Records</string>
            </property>
            <property name="styleSheet">
             <string notr="true">border-image: url(:/new/images/icons/view.png);</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
           <widget class="QLabel" name="label_2">
            <property name="styleSheet">
             <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;
color:red;</string>
            </property>
            <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;View Student Records&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
           </widget>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QSplitter" name="splitter_3">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <widget class="QPushButton" name="viewTable">
            <property name="styleSheet">
             <string notr="true">border-image: url(:/new/images/icons/table.png);

</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:12pt; font-weight:600; color:#ff0000;&quot;&gt;View Student Records(Table)&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
           </widget>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QSplitter" name="splitter_4">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <widget class="QPushButton" name="modifyStudent">
            <property name="font">
             <font>
              <family>MS Shell Dlg 2</family>
              <pointsize>10</pointsize>
              <weight>50</weight>
              <italic>false</italic>
              <bold>false</bold>
             </font>
            </property>
            <property name="mouseTracking">
             <bool>true</bool>
            </property>
            <property name="toolTip">
             <string>Modify Student Records</string>
            </property>
            <property name="styleSheet">
             <string notr="true">border-image: url(:/new/images/icons/modify.ico);</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:12pt; font-weight:600; color:#ff0000;&quot;&gt;Modify Student Records&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
           </widget>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QSplitter" name="splitter_5">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <widget class="QPushButton" name="deleteStudent">
            <property name="font">
             <font>
              <family>MS Shell Dlg 2</family>
              <pointsize>10</pointsize>
              <weight>50</weight>
              <italic>false</italic>
              <bold>false</bold>
             </font>
            </property>
            <property name="mouseTracking">
             <bool>true</bool>
            </property>
            <property name="toolTip">
             <string>Delete Student Records</string>
            </property>
            <property name="styleSheet">
             <string notr="true">border-image: url(:/new/images/icons/delete.png);</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:12pt; font-weight:600; color:#ff0000;&quot;&gt;Delete Student Records&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
           </widget>
          </widget>
         </item>
         <item>
          <widget class="QSplitter" name="splitter_6">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <widget class="QPushButton" name="changeStaffPass">
            <property name="font">
             <font>
              <family>MS Shell Dlg 2</family>
              <pointsize>10</pointsize>
              <weight>50</weight>
              <italic>false</italic>
              <bold>false</bold>
             </font>
            </property>
            <property name="mouseTracking">
             <bool>true</bool>
            </property>
            <property name="toolTip">
             <string>Change Password</string>
            </property>
            <property name="styleSheet">
             <string notr="true">
border-image: url(:/new/images/icons/change_password.ico);
</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:12pt; font-weight:600; color:#ff0000;&quot;&gt;Change Password&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
           </widget>
          </widget>
         </item>
         <item>
          <widget class="QSplitter" name="splitter_8">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <widget class="QPushButton" name="about">
            <property name="styleSheet">
             <string notr="true">border-image: url(:/new/images/Images/about_us.png);</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
           <widget class="QLabel" name="label_27">
            <property name="styleSheet">
             <string notr="true">color:red;

font: 75 14pt &quot;MS Shell Dlg 2&quot;;
</string>
            </property>
            <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;About Us&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
           </widget>
          </widget>
         </item>
         <item>
          <widget class="QSplitter" name="splitter_7">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <widget class="QPushButton" name="exitStaff">
            <property name="font">
             <font>
              <family>MS Shell Dlg 2</family>
              <pointsize>10</pointsize>
              <weight>50</weight>
              <italic>false</italic>
              <bold>false</bold>
             </font>
            </property>
            <property name="mouseTracking">
             <bool>true</bool>
            </property>
            <property name="toolTip">
             <string>Exit</string>
            </property>
            <property name="styleSheet">
             <string notr="true">border-image: url(:/new/images/icons/exit.png);</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:12pt; font-weight:600; color:#ff0000;&quot;&gt;Exit&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
           </widget>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
